src/components/toggle-button/toggle-button.stories.js

```diff
@@ -0,0 +1,172 @@
+    icon: {
+      name: 'data-vl-icon',
+      type: { name: TYPES.STRING, required: false },
+      description: 'Sets the icon of the toggle button.',
+      table: {
+        type: { summary: TYPES.STRING },
+        category: CATEGORIES.ATTRIBUTES,
+      },
+    },
+    iconPlacement: {
+      name: 'data-vl-icon-placement',
+      description: 'Determines where the icon should be rendered before or after the text.',
+      table: {
+        type: { summary: TYPES.STRING },
+        category: CATEGORIES.ATTRIBUTES,
+        defaultValue: { summary: ICON_PLACEMENT.AFTER },
+      },
+      control: {
+        type: 'select',
+        options: [ICON_PLACEMENT.BEFORE, ICON_PLACEMENT.AFTER],
+      },
+    },
+    textHidden: {
+      name: 'data-vl-text-hidden',
+      description: 'Determines whether the toggle button text is shown.',
+      table: {
+        type: { summary: TYPES.BOOLEAN },
+        category: CATEGORIES.ATTRIBUTES,
+        defaultValue: { summary: 'false' },
+      },
+    },
+    change: {
+      name: 'change',
+      description: 'Custom event that is triggered when the active state of the toggle button changes.',
+      table: { category: CATEGORIES.EVENTS },
+    },
+    click: {
+      name: 'click',
+      description: 'Custom event fired on click of the toggle button.',
+      table: { category: CATEGORIES.EVENTS },
+    },
+    ...argTypesShared,
+    error: {
+      ...argTypesShared.error,
+      description:
+        'Used to emphasize the importance or consequences of an action when the toggle button is in an active state.',
+    },
+  },
+};
+
+export const Default = ({
+  icon,
+  iconPlacement,
+  content,
+  textHidden,
+  error,
+  block,
+  large,
+  wide,
+  narrow,
+  loading,
+  disabled,
+  change,
+  click,
+}) =>
+  html`<vl-toggle-button
+    data-vl-icon=${ifDefined(icon)}
+    data-vl-icon-placement=${ifDefined(iconPlacement)}
+    ?data-vl-text-hidden=${textHidden}
+    ?data-vl-error=${error}
+    ?data-vl-block=${block}
+    ?data-vl-large=${large}
+    ?data-vl-wide=${wide}
+    ?data-vl-narrow=${narrow}
+    ?data-vl-loading=${loading}
+    ?disabled=${disabled}
+    @change=${(event) => change(event.detail)}
+    @click=${(event) => {
+      click(event);
+    }}
+    >${content}
+  </vl-toggle-button>`;
+
+// Get last toggle button, because storybook can render multiple stories
+const getToggleButton = () => {
+  const [lastItem] = [...document.querySelectorAll('vl-toggle-button')].slice(-1);
+  return lastItem;
+};
+
+export const Controlled = ({
+  active,
+  icon,
+  iconPlacement,
+  content,
+  textHidden,
+  error,
+  block,
+  large,
+  wide,
+  narrow,
+  loading,
+  disabled,
+  change,
+  click,
+}) => html`<vl-toggle-button
+  .active=${active}
+  data-vl-icon=${ifDefined(icon)}
+  data-vl-icon-placement=${ifDefined(iconPlacement)}
+  ?data-vl-text-hidden=${textHidden}
+  ?data-vl-error=${error}
+  ?data-vl-block=${block}
+  ?data-vl-large=${large}
+  ?data-vl-wide=${wide}
```

Comment:
Als ik de wide control aanzet en ik hover over de toggle button, gaat er iets mis.

Reply:
Da's een bug in button